styles/bsky/catppuccin.user.less

```
@@ -538,6 +538,9 @@
     div[style*="background-color: rgb(0, 17, 36)"], // dark
     div[style*="background-color: rgb(21, 41, 63)"], // dim
     div[style*="background-color: rgb(240, 247, 255)"] // light
+    a[style*="background-color: rgb(0, 17, 36)"], // dark
+    a[style*="background-color: rgb(21, 41, 63)"], // dim
+    a[style*="background-color: rgb(240, 247, 255)"] // light
```

```suggestion
   [style*="background-color: rgb(0, 17, 36)"], // dark
   [style*="background-color: rgb(21, 41, 63)"], // dim
   [style*="background-color: rgb(240, 247, 255)"] // light
```

How about just simplifying the selector like this?

---

would work, though i feel like keeping a wildcard selector specific to 2 types of elements (instead of *all*) is a better idea performance-wise (obviously only if done a lot + i haven't actually benchmarked that), im fine with either way though
(reasoning: i wanna avoid my styles becoming similar to the discord style in terms of performance)